### DISCUSSION TITLE
How do I change the color of the Tour Segmenter totals?

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1256] [General]
Try as I might, I cannot find the view color tool for the totals in the Tour Segmenter.
![color of totals](https://github.com/mytourbook/mytourbook/assets/106929478/051bbf5e-65bb-4e83-88f5-f54c18cd1149)

**Comment by wolfgang-ch** [comment]
The colors can be defined in this slideout

![image](https://github.com/mytourbook/mytourbook/assets/1283445/6d74a650-b5f0-4477-9363-6444ca0b9893)

**Comment by stvzldn** [comment]
Thank you for help, I had been looking at this control.
![color of totals View](https://github.com/mytourbook/mytourbook/assets/106929478/79e75028-c25d-41bc-a2aa-180c0614f4c1)
